internal/cmd/logout_test.go

```diff
@@ -127,4 +239,30 @@ func TestLogout(t *testing.T) {
 		assert.NilError(t, err)
 		assert.DeepEqual(t, expectedKubeCfg, updatedKubeCfg, cmpopts.EquateEmpty())
 	})
+
+	t.Run("with one and all", func(t *testing.T) {
+		testFields := setup(t)
+		err := Run(context.Background(), "logout", testFields.serverURLs[0], "--all")
+		assert.Error(t, err, "Argument [SERVER] and flag [--all] cannot be both specified.")
+
+		assert.Equal(t, int32(0), atomic.LoadInt32(testFields.count), "calls to API")
+
+		updatedCfg, err := readConfig()
+		assert.NilError(t, err)
+		assert.DeepEqual(t, &testFields.config, updatedCfg)
+	})
+
+	t.Run("error", func(t *testing.T) {
+		testFields := setupError(t)
+		err := Run(context.Background(), "logout", testFields.serverURLs[0])
+		assert.Error(t, err, fmt.Sprintf(`Failed to logout of server %s due to an internal error: POST "/v1/logout" responded 500: Internal Server Error.`, testFields.serverURLs[0]))
```

Comment:
It would probably be ok to be less strict about the error here, and use something like this:

```suggestion
		assert.ErrorContains(t, err, "Failed to logout of server "+testFields.serverURLs[0])
```